src/main/java/github/weichware10/analyse/Diagramm.java

```diff
@@ -0,0 +1,67 @@
+package github.weichware10.analyse;
+
+import github.weichware10.util.Data;
+import java.util.List;
+
+/**
+ * verantwortlich für die Erstellung der Diagramm-Diagramme.
```

Comment:
Soll es Diagramm-Diagramme heißen? Falls ja, einfach ignorieren

Reply:
Genauso in Zeile 14

Reply:
Ja, das Diagramm soll für die Diagramm-Analysetypen stehen